sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLParserSuite.scala

```diff
@@ -184,6 +184,15 @@ class DDLParserSuite extends AnalysisTest with SharedSQLContext {
       containsThesePhrases = Seq("key_without_value"))
   }
 
+  test("alter database set location") {
+    // ALTER (DATABASE|SCHEMA) database_name SET LOCATION
+    val sql1 = "ALTER DATABASE database_name SET LOCATION '/home/user/db'"
```

Comment:
Add negative cases? For example, set an illegal uri string?

Reply:
Add it to DDLSuite?

Reply:
I add a positive e2e test case.
What's the negative test case you want to add ?

Reply:
1. A database that does not exist before issuing this command?
2. Setting an illegal path that could trigger an exception inside the function `stringToURI`